**Exercício S10:** 

Verificar se a altura média da turma é significativamente diferente de 161 cm

**Dados da Amostra:**

    Altura da turma (em cm): [165, 157, 163, 170, 167, 161, 160, 170, 157, 160, 157, 168], 162, 170, 167, 165, 166, 158, 174, 169, 163, 174, 162, 170, 155, 154, 158, 160, 166, 160]

**Hipótese Nula (H0):**

Não há diferença entre a média da amostra e a média da população (161 cm). A média da altura das alunas dessa turma é 161 cm.

**Hipótese Alternativa (H1):**

Existe uma diferença entre a média da amostra e a média da população. A média da altura das alunas dessa turma não é 161 cm.

TesteZ

In [2]:
import numpy as np 
from statsmodels.stats.weightstats import ztest
altura_on_35 = np.array([165, 157, 163, 170, 167, 161, 160, 170, 157, 160, 157, 168, 162, 170, 167, 165, 166, 158, 174, 169, 163, 174, 162, 170, 155, 154, 158, 160, 166, 160])
media_nacional = 161

z_statistic, p_value = ztest(altura_on_35, value= media_nacional)
print("Teste Z para uma amostra:")
print(f"z-statistic: {z_statistic:4f}, p-value: {p_value:.4f}")

Teste Z para uma amostra:
z-statistic: 2.568889, p-value: 0.0102


**Análise dos resultados:**

Estatística Z: 2.56889
Valor p: 0.0102

Dado que o nível de significância é 0,05 e o p-value obtido é 0.0102, rejeitamos a hipótese nula. A média da altura das alunas da turma On 35 é diferente da média nacional de 161 cm.